scene_graph/download_full_data.sh

```diff
@@ -0,0 +1,37 @@
+# Execute from snorkel-tutorials/
+# Download data,
+
+ANNOTATIONS_URL="https://www.dropbox.com/s/bnfhm6kt9xumik8/vrd.zip"
+IMAGES_URL="http://imagenet.stanford.edu/internal/jcjohns/scene_graphs/sg_dataset.zip"
+SAMPLE_IMAGES_URL="https://github.com/Prof-Lu-Cewu/Visual-Relationship-Detection.git"
+GLOVE_URL="http://nlp.stanford.edu/data/wordvecs/glove.6B.zip"
+
+if [ ! -d "scene_graph/data" ]; then
+    mkdir -p scene_graph/data
+    cd scene_graph/data
+
+    # download and unzip metadata and annotations
+    wget $ANNOTATIONS_URL
+    unzip vrd.zip
+
+    # Delete the zip files.
+    rm vrd.zip
+    cd VRD
+
+    # Download and unzip all images
+    wget $IMAGES_URL
+    unzip sg_dataset.zip
+    rm sg_dataset.zip
+    cd ../../..
```

Comment:
nit: is there a way to run all these commands from the root directory (e.g. `unzip something/something/sg_dataset.zip`) so we aren't `cd ../../..`? might look cleaner

Reply:
I tried that originally and I couldn't get the unzip results to land up in the correct location somehow, will make a note for later though